samples/spring-boot-security/src/main/java/sample/Application.java

```diff
@@ -2,57 +2,45 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
-import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.provisioning.InMemoryUserDetailsManager;
+import org.springframework.security.web.SecurityFilterChain;
 
 @SpringBootApplication
 public class Application {
 
     @Configuration
-    protected static class ApplicationSecurity extends WebSecurityConfigurerAdapter {
+    protected static class ApplicationSecurity {
 
-        @Override
-        protected void configure(HttpSecurity http) throws Exception {
+        @Bean
+        protected SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
             // @formatter:off
             http
                     .authorizeRequests()
-                        .anyRequest()
-                        .authenticated()
-                        .and()
+                    .anyRequest()
```

Comment:
Could you please revert formatting in this block to the keep original one as it was requested by its author @bennetelli using @formatter:off hint ?